ern-container-gen-android/src/hull/lib/src/main/java/com/walmartlabs/ern/container/ElectrodeMiniAppFragment.java

```diff
@@ -44,4 +52,14 @@ public void onAttach(Context context) {
                     + " must implement ElectrodeReactActivityListener");
         }
     }
-} 
\ No newline at end of file
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        if (mMiniAppView instanceof ReactRootView) {
+            mElectrodeReactActivityListener.getElectrodeDelegate()
+                    .removeMiniAppView(getMiniAppName(), (ReactRootView) mMiniAppView);
```

Comment:
Now that I'm thinking about this, do we properly cleanup views for PML use cases?

Reply:
yes, we are not using `ElectrodeReactActivityDelegate`, we are calling `rootView.startReactApplication` directly.